pkg/beacon/repository.go

```diff
@@ -50,6 +50,7 @@ type Result struct {
 	data.UniqueIPPair `bson:",inline"`
 	Connections       int64   `bson:"connection_count"`
 	AvgBytes          float64 `bson:"avg_bytes"`
+	TB								int64    `bson:"total_bytes"`
```

Comment:
Looks like indention is a bit off here. Guessing a tabs/spaces issue?

Reply:
Would you be opposed to expanding TB out into TotalBytes? Just to cut down on the number of acronyms. Not a big deal, just a suggestion.

Reply:
TotalBytes is taken. I'm not too familiar with go, but I assume I cannot reuse the variable name even if it's within a struct. I can definitely expand it though.

Reply:
Sorry, can you please clarify a bit? As long as there isn't another struct field with the same name in that struct then you should be able to use that name.

Reply:
No you're right I just got confused. Everything mentioned above has been changed. :)